.github/workflows/macOS-tests.yml

```diff
@@ -39,6 +39,10 @@ jobs:
           pip3 install --upgrade pip
           pip3 install codespell
           codespell --count --ignore-words-list=ans,deriver,inout,packag --skip="*.js,*WordLists.swift"
+      - name: SwiftLint
+        run: |
+          swiftlint --fix
```

Comment:
Was the reason for the `--fix` flag to make the CI prompt only about the errors that cannot be fixed automatically?

Reply:
Yes.

Reply:
Added a comment about it in CI.